embeddable-say-editor/main.js

```diff
@@ -163,6 +171,9 @@ export async function renderEditor({
         const { height: contentHeight } = entry.contentRect;
 
         editorFrame.style.height = `${contentHeight}px`;
+        //tried a bunch of values, 100px worked the best, no idea why
+        const MAGIC_OFFSET = '100px';
```

Comment:
92px is the height of the toolbar + the padding of the paper, so this would seem like a sensible number. I'm not sure why 100px works without causing the infinite growing problem...